### DISCUSSION TITLE
Importing audio support / sound effects

### DISCUSSION THREAD

**Discussion by theEpicjosh** [Orama-Interactive/Pixelorama#897] [Ideas]
Can you support importing audio implementation such as sound effects or music for making full animations or prototypes.
Feel free to ask me to elaborate further if you need me to!

**Comment by Variable-ind** [comment]
i made an extension that allows for the existence of audio as a "Reference track" **(meaning it will not be available in export and can only be used to get an idea of sound with correspondence to animation)**

here's the extension if you wish to try it (For pixelorama 0.11.1)
(unzip it, drag and drop to pixelorama, and enable it from Preferences>Extensions ) [Audia.zip](https://github.com/Orama-Interactive/Pixelorama/files/12362467/Audia.zip)

https://youtu.be/Znj-lzboRqQ

**Comment by OverloadedOrama** [comment]
Implemented by #1149.
